.github/workflows/ci-workflow.yml

```diff
@@ -234,6 +248,16 @@ jobs:
           java-version: '8'
       - name: Build with Maven
         env:
-          MAVEN_OPTS: -Xmx2g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN -Dmaven.surefire.arguments="-Duser.language=fr -Duser.region=FR -Duser.timezone=Europe/Paris" -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
+          MAVEN_OPTS: -Xmx2g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN -Duser.language=fr -Duser.country=FR -Duser.region=FR -Duser.timezone=Europe/Paris -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
         run: |
           mvn -V -T 0.7C package -B -Ddir-only -ntp -ff -pl -nifi-assembly -pl -nifi-system-tests -nsu
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        if: ${{ false }}
+        with:
+          name: surefire-reports-fr
+          path: |
+            '**/surefire-reports/TEST-*.xml'
+            '!**/node_modules/**'
+            '!**/nifi-nar-bundles/nifi-standard-bundle/nifi-jolt-transform-json-ui/**'
```

Comment:
https://github.com/greyp9/nifi/actions/runs/793436719

`
Windows Zulu JDK8 FRNo files were found with the provided path: '**/surefire-reports/TEST-*.xml' '!**/node_modules/**' '!**/nifi-jolt-transform-json-ui/**'. No artifacts will be uploaded.
`

This output suggests that use of exclusion patterns may be a viable workaround.